R/submit-models.R

```diff
@@ -128,6 +128,11 @@ submit_nonmem_models <- function(.mods,
         cmd_args,
         sprintf("--config=%s", normalizePath(.config_path))
       )
+    } else {
+      default_config_path <- file.path(model_dir, "bbi.yaml")
+      if (!fs::file_exists(default_config_path) && !isTRUE(.dry_run)) {
+        stop(glue("Looking for default configuration file at {default_config_path}: no such file or directory"), call. = F)
```

Comment:
Hmm, why does the error message say "or directory"?  Aren't we checking for and expecting a regular file?